examples/django_app/requirements.txt

```diff
@@ -1,2 +1,4 @@
 django>=1.8.0,<2.0.0
 chatterbot>=0.5.0
+gunicorn
+whitenoise
```

Comment:
If the example_app is going to use web service/hosting definitely. If web serive it won't consider static files, If we want serve static files then we have to use ``whitenoise`` module.

@gunthercox There is other way, I'll remove the changes from here and add these steps in ``django.rst`` documentation page.